src/ergw_aaa_nasreq.erl

```diff
@@ -541,3 +548,58 @@ create_STR(Session, Opts) ->
 
 get_state_atom(#state{state = State}) ->
     State.
+
+handle_common_request(Command, #{'Session-Id' := SessionId} = Avps, {_PeerRef, Caps}) ->
+    {Result, ReplyAvps0} =
+	case ergw_aaa_session_reg:lookup(SessionId) of
+	    Session when is_pid(Session) ->
+		ergw_aaa_session:request(Session, ?MODULE, {?API, Command}, Avps);
+	    _ ->
+		{{error, unknown_session}, #{}}
+	end,
+
+    #diameter_caps{origin_host = {OH,_},
+		   origin_realm = {OR,_},
+		   origin_state_id = {OSid, _}} = Caps,
+
+    ReplyAvps1 = filter_reply_avps(Command, ReplyAvps0),
+    ReplyAvps2 =
+	ReplyAvps1#{'Origin-Host' => OH,
+		    'Origin-Realm' => OR,
+		    'Origin-State-Id' => OSid,
+		    'Session-Id' => SessionId},
+    ReplyCode = diameter_reply_code(Command),
+    ReplyAvps = diameter_reply_avps(Result, ReplyAvps2),
+    ?LOG(error, "~p reply Avps: ~p", [Command, ReplyAvps]),
+    {reply, [ReplyCode | ReplyAvps]}.
+
+filter_reply_avps('RAR', Avps) ->
+    Permited =
+	['OC-OLR', 'IP-CAN-Type', 'RAT-Type',
+	 'AN-Trusted', 'AN-GW-Address',
+	 '3GPP-SGSN-MCC-MNC', '3GPP-SGSN-Address', '3GPP-SGSN-IPv6-Address',
+	 'RAI', '3GPP-User-Location-Info', 'User-Location-Info-Time',
+	 'NetLoc-Access-Support', 'User-CSG-Information',
+	 '3GPP-MS-TimeZone', 'Default-QoS-Information',
+	 'Charging-Rule-Report'],
+    maps:with(Permited, Avps);
+filter_reply_avps('ASR', Avps) ->
+    Permited = ['User-Name'],
```

Comment:
this need to checked as well.

Reply:
Based on https://tools.ietf.org/html/rfc4005#section-3.7 and https://tools.ietf.org/html/rfc4005#section-3.3 the `ASR` and `RAR` has AVP `User-Name`.

Reply:
Updated based on `rfc4005`:
* https://tools.ietf.org/html/rfc4005#section-3.4
* https://tools.ietf.org/html/rfc4005#section-3.8